src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+
+        String action = null;
+        Scanner input = new Scanner(System.in);
+        Task[] allTasks = new Task[100];
+        int numTasks = 0;
+        int taskNum = 0;
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("----------------------------------------------------");
+        System.out.println(logo);
+        System.out.println("Aloha! I'm Duke");
+        System.out.println("What would you like to do today?");
+        System.out.println("Enter 'Bye' when done.");
+        System.out.println("----------------------------------------------------");
+
+        // obtain string input
+        if(!input.hasNext() || (action = input.nextLine()).isEmpty()) System.exit(0);
```

Comment:
Conditional should be on separate line and should wrap single statement in curly brackets (based on module's coding standards)